react-front-end/tsrc/components/ShuffleBox.tsx

```diff
@@ -161,7 +161,7 @@ export const ShuffleBox = ({
 
   return (
     <Grid id={id} container spacing={2}>
-      <Grid item xs={3}>
+      <Grid item xs>
```

Comment:
Do you need `xs` at all in this case? Can it just be:

```suggestion
      <Grid item>
```

(Same query for other one below too.)

Reply:
I had the same idea before but unfortunately, we need this. :smile: 
https://mui.com/components/grid/#auto-layout